### PR TITLE
[15.0][IMP] stock_picking_batch_extended: batch relation in stock move

### PR DESCRIPTION
**Pull request by chienandalu**
Please check @carlosdauden @sergio-teruel 

Add the relation to the batch picking in the stock.move so we can filter / group by that criteria.

cc @Tecnativa TT47127

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

File: stock_picking_batch_extended/migrations/15.0.3.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def _set_move_id_picking_batch(env):
+    """Pre-fill column"""
+    openupgrade.logged_query(
```
Comment (pedrobaeza) [inline stock_picking_batch_extended/migrations/15.0.3.0.0/pre-migration.py]: This script is not needed, as now related fields are pre-filled with a similar SQL to this one.

Comment (chienandalu) [reply stock_picking_batch_extended/migrations/15.0.3.0.0/pre-migration.py]: Ok, I just removed it :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Comment by rousseldenis** [comment]
@chienandalu Maybe an index on it should have been great

**Pull request merged**